### PR TITLE
Set up boilerplate

### LINKED ISSUE #78: Add Elm/Postcss boilerplate to repo
* Add Elm
* Add PostCSS
* Add backend

### PR DESCRIPTION
**Pull request by ivanmauricio**
* Added boilerplate with Elm and postcss
* Scaffold out database files

resolves #78 
relates #102

### CODE REVIEW

File: tests/server/index.test.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        if (err) {
+          done(err);
+        }
+        expect(res.statusCode).to.equal(200);
+        expect(res.type).to.equal('application/javascript');
+        done();
+      });
+  });
+});
+//
```
Comment (sophielevens) [inline tests/server/index.test.js]: Hey! Can you delete the commented out code here?

File: src/elm/Helpers.elm [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module Helpers exposing (..)
```
Comment (sophielevens) [inline src/elm/Helpers.elm]: Weren't there other helpers in here?

File: tests/server/index.test.js (hunk 1 above)
Comment (ivanmauricio) [reply tests/server/index.test.js]: @sophielevens Done!

**Comment by ivanmauricio** [comment]
@sophielevens Thanks for thorough code review 👁 Have made changes now!

**Pull request merged**